src/edutap/wallet_apple/models/passes.py

```diff
@@ -494,17 +717,19 @@ def barcode(self, value: Barcode | None):
     This key works only for poster event tickets.
     """
 
-    relevantDate: str | DateField | None = None  # Deprecated: use relevantDates instead
+    relevantDate: str | None = pydantic.Field(
+        default=None, deprecated="Use relevantDates instead"
+    )
     """
     Optional.
     The date and time when the pass becomes relevant, as a W3C timestamp, such as the start time of a movie. The value needs to be a complete date that includes hours and minutes, and may optionally include seconds.
     For information about the W3C timestamp format, see Time and Date Formats on the W3C website.
     This object is deprecated. Use relevantDates instead.
     """
 
-    relevantDates: list[DateField] | None = None
+    relevantDates: list[RelevantDate] | None = None
     """
-    Optional.
+    Optional. [Pass.RelevantDates]
```

Comment:
I do not think this should be here
```suggestion
    Optional. 
```